airflow/cli/commands/scheduler_command.py

```diff
@@ -74,6 +57,26 @@ def scheduler(args):
         signal.signal(signal.SIGQUIT, sigquit_handler)
         _run_scheduler_job(args=args)
 
+def _run_scheduler_job(args):
```

Comment:
Just curious - why move these functions?

Reply:
Great point! Did intend to, so I just undid the reordering of those methods